maxtrix.js

```diff
@@ -1,16 +1,28 @@
 
-function makeArray(rows, columns){
+function make2DArray(rows, columns, fill){
   let matrix = [];
   for (var i = 0; i < rows; i++) {
       matrix.push([]);
       for (var j = 0; j < columns; j++) {
-          matrix[i].push([j+1]);
+          matrix[i].push([fill]);
       }
   }
 
   return matrix;
+}
+
+firstArray = make2DArray(5, 10, "ham");
 
-  console.log(matrix);
+function expand2DArray(firstArray, rows, columns, fill){
```

Comment:
One thing you can do it make this all one function. Change the method signature to check if an array is passed. if it is I modifies it, else it create new.

`make2DMatrix(rows, columns, fill, array)`